src/config/ConfigMetadata.ts

```diff
@@ -296,6 +304,10 @@ export const ConfigMetadata: {[key: string]: { type: string, defaultValue?: any
         type: 'boolean',
         defaultValue: false,
     },
+    [ConfigNames.THUNDRA_TRACE_INTEGRATIONS_GOOGLE_SUBSCRIPTION_DISABLE]: {
```

Comment:
`THUNDRA_TRACE_INTEGRATIONS_GOOGLE_PUBSUB_DISABLE` to disable both tracing publish and subscribe operations